ruby/example_code.rb

```diff
@@ -7,20 +7,17 @@
   :startup,
   :my_event,
   :my_other_event
-]
+].freeze
 
-log = MicroserviceLogger.new(:service_name => 'my_super_service',
-                             :clock => Time,
-                             :output => $stdout,
-                             :events => MY_EVENTS
-                            )
```

Comment:
Sadly I don't see a lot of hash rockets these days, the Rubocop default is definitely in line with common practice in this case.